cfme/scripting/appliance.py

```diff
@@ -29,6 +33,56 @@ def main():
     pass
 
 
+@main.command('upgrade', help='Upgrades an appliance to new version')
+@click.argument('appliance-ip', default=None, required=False)
+@click.option('--cfme-only', is_flag=True, help='Upgrade cfme packages only')
+@click.option('--update-to', default='59', help='Must be 58 or (59 is default)')
```

Comment:
**REQUIRED** This will have to accept version, not just '58' or '59' string. The moment you show this to the public, they will be asking `I have this BZ that requires DB migrated to 5.8.3.1, how do I do that?` which means we will have to support that.
Let's start by changing these `58` and `59` strings to `5.8.z` and `5.9.z` so that it's clear we update to latest of that stream.

Next step (future PR) would be to be able to `--list-supported-versions` to get a list of versions the given appliance (based on it's version) can be updated to. This also depends on our yamls supporting that by having proper repo files available, not just the latest builds.